src/main/signal_handlers.cpp

```diff
@@ -71,11 +84,7 @@ void* cvc4StackBase;
 #endif /* HAVE_SIGALTSTACK */
 
 /** Handler for SIGXCPU and SIGALRM, i.e., timeout. */
-void timeout_handler(int sig, siginfo_t* info, void*) {
-  safe_print(STDERR_FILENO, "CVC4 interrupted by timeout.\n");
-  print_statistics();
-  abort();
-}
+void timeout_handler(int sig, siginfo_t* info, void*) { timeout_handler(); }
```

Comment:
It feels like it would be more consistent to move this function to src/main/time_limit.cpp (similar to `win_timeout_handler()`).